### PR TITLE
[@wordpress/components] Add AnglePickerControl

### PR DESCRIPTION
**Pull request by sarayourfriend**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/WordPress/gutenberg/blob/d20270f0d40955b46378f9988e5256a2a06a27c3/packages/components/src/angle-picker-control/README.md
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (I don't think it's necessary to do this as AnglePickerControl was added before the version number).
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wordpress__components ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wordpress__components@latest/))
was missing the following properties:
1. CustomSelectControl
1. Guide
1. GuidePage
1. TextHighlight
1. Tip

<details>
<summary>as well as these 2 other properties...</summary>
<p>ToolbarGroup, VisuallyHidden</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against bdec483a6407620071dc0e2851aabfb387fc70ca
</p>

File: types/wordpress__components/angle-picker-control/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    interface Props {
+        /**
+         * The current value of the input. The value represents an angle in degrees and should be a value between 0 and 360.
+         */
+        value: number;
+
+        /**
+         * Label to use for the angle picker. If not set the a translated label "Angle" is used.
+         */
+        label: string;
```
Comment (sirreal) [inline types/wordpress__components/angle-picker-control/index.d.ts:13]: [Docs say this isn't required](https://github.com/WordPress/gutenberg/blob/aa0abb915e843e2d636d4242aa249e504284abbc/packages/components/src/angle-picker-control/README.md#label), but I haven't confirmed (it's just passed to `BaseControl`).

```suggestion
        label?: string;
```

**Review by sirreal — COMMENTED** [review]
One note, thanks!

**Review by sirreal — APPROVED** [review]

**Comment by sirreal** [comment]
Ready to merge

**Pull request merged**